ChangeLog

```diff
@@ -73,7 +73,7 @@ ver. 1.0.1 (2022/09/27) - energy-equals-mass-times-the-speed-of-light-squared
 * [stability] solves race condition with uncontrolled growth of failure list (jail with too many matches,
   that did not cause ban), behavior changed to ban ASAP, gh-2945
 * fixes search for the best datepattern - e. g. if line is too short, boundaries check for previously known
-  unprecise pattern may fail on incomplete lines (logging break-off, no flush, etc), gh-3020
```

Comment:
Is it OK to modify the `Changelog`?

Reply:
Sure, why not, just ...
I don't think it is justified in that case: `unprecise` is a fully valid synonym for `imprecise` and even [OED contains it](https://www.oed.com/search/dictionary/?q=unprecise).

Reply:
Ah, indeed. not sure how it ended up in the list of typos. I wasn't aware the OED is still available online, that's good to know. As for `unprecise`, it is missing from other online dictionaries except [Merriam-Webster](https://www.merriam-webster.com/dictionary/unprecise):
* [Oxford Learner's](https://www.oxfordlearnersdictionaries.com/spellcheck/english/?q=unprecise)
* [Cambridge](https://dictionary.cambridge.org/fr/spellcheck/anglais/?q=unprecise)
* [Collins](https://www.collinsdictionary.com/spellcheck/english?q=unprecise)
* [Longman](https://www.ldoceonline.com/spellcheck/english/?q=unprecise)
* [Merriam-Webster](https://www.merriam-webster.com/dictionary/unprecise)

However, the Google Ngram Viewer shows that `unprecise` is hardly used, whether in American or British English:
https://books.google.com/ngrams/graph?content=unprecise%2Cimprecise&year_start=1900&year_end=2019&corpus=en-2019